dataland-frontend/src/components/forms/parts/fields/StringBaseDataPointFormField.vue

```diff
@@ -41,12 +46,15 @@ export default defineComponent({
       documentReference: undefined as string | undefined,
       fileNamesForPrefill: [] as string[],
       isMounted: false,
+      isValidFileName: isValidFileName,
     };
   },
   emits: ["fieldSpecificDocumentsUpdated"],
   mounted() {
-    this.updateFileUploadFiles();
-    this.isMounted = true;
+    setTimeout(() => {
```

Comment:
This is to execute this only in the next event cycle? Can we talk about this?

Reply:
Not my code.
Git says @pioSko2

Reply:
Yes, mine. setTimeout is needed to create enough time for the page to load and populate dataSource inputs before the dataSource form is hidden. Without it, when editing, the dataSource is not filled in and re-sent empty

Reply:
Will gladly discuss if you have another solution

Reply:
That is good to know. I fear we have the same issue on other form elements. Should we open a ticket regarding that?

Reply:
All inputs with documents have been checked and changed in DALA-3036. Other inputs seemed to work fine.
However, if you feel there may be other areas where we should check then sure. I'd be happy to test that, too.

Reply:
This was specific for dataSource/documents

Reply:
My bad. I somehow did not notice it at the other places, yet.